src/Command/LintTemplatesCommand.php

```diff
@@ -509,6 +525,23 @@ private function assertValidBooleanProperty(string $property, array $attribute,
         }
     }
 
+    private function assertValidAttributeValidationRule(array $attribute, int $index, string $fileName, array $violations): void
+    {
+        $propertyPath = sprintf('[attributes][%d][validation_rule]', $index);
+        switch (true) {
+            case (!empty($attribute['validation_rule']) && !in_array($attribute['validation_rule'], self::VALIDATION_RULES)):
+                $violations[$fileName]->add(new ConstraintViolation(
+                    'This value is not a valid validation rule.',
```

Comment:
Since the repository became public, would it be reasonable to add to the exception message also the list of allowed validation rules - to simplify the adjustments for the original `.json` file?